<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>911</width>
     <height>631</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>576</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>입학년도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fresh_year">
        <property name="minimum">
         <number>1986</number>
        </property>
        <property name="maximum">
         <number>2022</number>
        </property>
        <property name="value">
         <number>2022</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>학과</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Major"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>목표학점</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="goal_grade">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>4.300000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="person_info">
        <property name="text">
         <string>apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListView" name="recommend_view">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>151</y>
       <width>171</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="load_view">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>271</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="good_grade_view">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>150</y>
       <width>271</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>500</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
